flutter_common/lib/src/utils/env_loader.dart

```diff
@@ -13,8 +13,13 @@ String envFilePath() {
 
 Future<void> loadEnv() async {
   await dotenv.load(fileName: envFilePath());
-  final supabaseUrl = dotenv.env['STUDYU_SUPABASE_URL']!;
-  final supabaseAnonKey = dotenv.env['STUDYU_SUPABASE_PUBLIC_ANON_KEY']!;
+  final supabaseUrl = dotenv.env['STUDYU_SUPABASE_URL'] ?? const String.fromEnvironment('STUDYU_SUPABASE_URL');
+  final supabaseAnonKey =
+      dotenv.env['STUDYU_SUPABASE_PUBLIC_ANON_KEY'] ?? const String.fromEnvironment('STUDYU_SUPABASE_PUBLIC_ANON_KEY');
+  final envAppUrl = dotenv.env['STUDYU_APP_URL'] ?? const String.fromEnvironment('STUDYU_APP_URL');
+  final envDesignerUrl = dotenv.env['STUDYU_DESIGNER_URL'] ?? const String.fromEnvironment('STUDYU_DESIGNER_URL');
+  final envProjectGeneratorUrl =
+      dotenv.env['STUDYU_PROJECT_GENERATOR_URL'] ?? const String.fromEnvironment('STUDYU_PROJECT_GENERATOR_URL');
```

Comment:
Before these changes this would have crashed if an env variable was missing (where the optional `dotenv.env[...]` was force unwrapped), now the value is set to an empty string (`String.fromEnvironment('I_DONT_EXIST') == ""`), i.e. we probably crash at some later point. Is this not a problem?

If it is we could use something like this

```dart
String? getEnv(String name) {
      return (dotenv.env[name] ?? (bool.hasEnvironment(name) ? String.fromEnvironment(name) : null))
}
```

And force-unwrap it to get the same behavior as before

Reply:
In both ways the end result would probably be: "Crash, without any meaningful error message for the user, because this is still a TODO". But alright let's take the changes to keep the bad crash behavior consistently bad. :D